main.py

```diff
@@ -151,6 +153,9 @@ def get_all_devices(self) -> list[Device]:
             logging.info("Refreshing all devices cache")
             self._devices_cache = [Device(x) for x in self.api.list_devices_detailed()]
 
+            for device in self._devices_cache:
+                device.description = self._device_descriptions.get(device.id, None)
```

Comment:
The second parameter for `dict.get()` is `None`, there is no really need to set it explicitly.

Reply:
duh. thx